callbacks.go

```diff
@@ -4,38 +4,22 @@ import (
 	"time"
 
 	"github.com/Nitro/sidecar-executor/container"
-	"github.com/mesos/mesos-go/api/v0/executor"
-	mesos "github.com/mesos/mesos-go/api/v0/mesosproto"
+	mesos "github.com/mesos/mesos-go/api/v1/lib"
 	"github.com/relistan/go-director"
 	log "github.com/sirupsen/logrus"
 )
 
-// Callbacks from the Mesos driver. These are required to implement
```

Comment:
These were callbacks that never really were in use. I removed them and the logging just happens in the driver.